### ISSUE TITLE
Instead of Sleep'ing, can the tape 'auto-advance' upon completion of the previous command?

### ISSUE THREAD

**Issue by paragchandra** [charmbracelet/vhs#537]
**Is your feature request related to a problem? Please describe.**
I've started using VHS to record demos of command-line utilities I've been developing, which has been working pretty well - so first off, thanks for this great utility. I guess the main 'complaint' I have is that I have to repeatedly try a command to see how long it might take in the worst case, so that I know how long to Sleep for, so that VHS captures all the output of the command.

**Describe the solution you'd like**
I think it would be more intuitive if VHS could just wait for a command to complete - and optionally exit on nonzero return code - before proceeding to the next command. Maybe that can't be directly supported by the existing 'Type' command, since that could be just to type text into a text editor or similar, so perhaps a different keyword would be needed.

**Describe alternatives you've considered**
I've looked at `asciinema`, which does operate in this sort of 'blocking' manner I've described, but I _really_ like the ability to essentially 'script' my demos ahead of time and then let VHS take care of the rest, rather than typing live while `asciinema` records.

**Additional context**
Hopefully the description above is clear enough, and if this is 'by design', that's fine too..

**Comment by tobihans** [comment]
I agree with this too. If core team could guide me on what would need to be done, I'm more than happy to try a PR.

**Comment by pmonks** [comment]
Pretty sure this is (yet another) duplicate of issue #70, which has a fix (PR #257) which hasn't been released yet (issue #550).

**Comment by caarlos0** [comment]
Hey!

This should now have been fixed on latest VHS by using the `Wait` keyword: https://github.com/charmbracelet/vhs/releases/tag/v0.9.0
